python/docs/basics.md

```diff
@@ -0,0 +1,147 @@
+# RMM - the RAPIDS Memory Manager
```

Comment:
My concern hereis that this duplicates the content in the readme, so we have two places to edit this content. Perhaps just change the readme to a summary and link to this doc.

Reply:
Agree that we shouldn't duplicate. Will remove from README after we get the docs published on rapids.ai